websites/B/Baseball-Reference/presence.ts

```diff
@@ -0,0 +1,94 @@
+const presence = new Presence({
+		clientId: "1049610101725605899",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let team: string = null,
+	image = document.images.item(1).src;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		startTimestamp: browsingTimestamp,
+	};
+	presence.setActivity(presenceData);
+	if (document.location.hostname === "baseball-reference.com") {
+		if (document.location.pathname === "")
+			presenceData.details = "Viewing home page";
+	} else if (document.location.pathname.includes("/players")) {
+		presenceData.details = "Viewing a Player:";
+		presenceData.state = `${
+			document.querySelector("div#meta div h1 span").textContent
+		} | ${document
+			.querySelector("div#meta div p strong")
+			.nextSibling.textContent.replace(/\s/g, " ")
+			.trim()}`;
+		if (!presence.getActivity().largeImageKey)
+			presenceData.largeImageKey = image;
+
+		presenceData.buttons = [{ label: "View Player", url: document.URL }];
+
+		if (
+			document.querySelector(
+				"#meta > div:nth-child(2) > p:nth-child(5) > a"
+			) !== null
+		) {
+			team = document.querySelector(
+				"#meta > div:nth-child(2) > p:nth-child(5) > a"
+			).textContent;
+			presenceData.state = `${presenceData.state}\nFor The ${team}`;
```

Comment:
I'm not sure if newline characters work in presence states.

Reply:
> I'm not sure if newline characters work in presence states.

It doesn't work, just tested it.